index.html

```diff
@@ -0,0 +1,235 @@
+            <span id="health" class="badge badge-danger">20</span><br />
+            <span class="dark-text">Hunger: </span>
+            <span id="hunger" class="badge badge-warning">20</span><br />
+          </div>
+        </div>
+      </div>
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Settings</h3>
+          <div class="card-body">
+            <form role="form">
+              <div class="form-group">
+                <label class="dark-text">Server address</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="server"
+                  placeholder="mc.hackclub.com"
+                />
+              </div>
+              <div class="form-group">
+                <label for="exampleInputPassword1" class="dark-text"
+                  >Port</label
+                >
+                <input
+                  type="number"
+                  class="form-control"
+                  id="port"
+                  placeholder="25565"
+                />
+              </div>
+              <button
+                type="button"
+                class="btn btn-outline-primary"
+                onclick="setServer()"
+              >
+                Set
+              </button>
+            </form>
+          </div>
+        </div>
+      </div>
+      <!--
+      <div class="botCard">
+        <div class="card bg-light">
+          <div class="card-header">
+            <div class="botHead">
+              <h3 class="mb-0">
+                nickbot &ensp;<span class="badge badge-warning">In Use</span>
+              </h3>
+            </div>
+            <div class="botStartStop">
+              <div
+                class="btn-group btn-group-sm"
+                role="group"
+                aria-label="Basic example"
+              >
+                <button type="button" class="btn btn-success">
+                  <i data-feather="play"></i>
+                </button>
+                <button type="button" class="btn btn-danger" disabled>
+                  <i data-feather="pause"></i>
+                </button>
+              </div>
+            </div>
+          </div>
+          <div class="card-body">
+            <div class="row">
+              <div class="col-sm">
+                <h5 class="card-title">Actions:</h5>
+              </div>
+
+              <div class="col-sm">
+                <span class="dark-text">Coordinates:</span>
+
+                <span id="coords" class="badge badge-primary">0 0 0</span><br />
+
+                <span class="dark-text">Health: </span>
+                <span id="health" class="badge badge-danger">20</span><br />
+                <span class="dark-text">Hunger: </span>
+                <span id="hunger" class="badge badge-warning">20</span><br />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    -->
```

Comment:
What's the role of the commented code here? Is this necessary to explain something or is it just unused?

If the latter, let me know and I can remove it.